### ISSUE TITLE
Module request equality

### ISSUE THREAD

**Issue by legendecas** [tc39/proposal-module-sync-assert#1]
Import attributes are recognized as part of the module equality comparison. When a module is imported twice, one with sync assertion and one without sync assertion, the module should only be evaluated once regardless of the assertion presented. However, if the assertion was part of the import attributes, two import requests would resolve to two distinguished modules.

**Comment by ljharb** [comment]
While I agree with this, I suspect browsers won't be able to guarantee that - iow, if you fire off two `import()`s at the same time, and nondeterministically, one resolves first, then they wouldn't be able to guarantee which one would error.

**Comment by Jack-Works** [comment]
> Import attributes are recognized as part of the module equality comparison.

Yes, so it no longer is a valid solution. It used to be cache-independent (import assert)
